nss-day-cohort-45/tabloidfullstack-giant-gibbons
# Gt tag crud

## Changes Made:
1. Tags can now be deleted
2. Tags can now be edited
## How to Check:
1. fetch gt-tagCRUD
2. Log in as an admin and navigate to Tag Management
3. Click on the Edit button on a tag to be taken to the edit form
4. Change the tag's name and hit Save; verify that the tag was edited
5. Click on the Delete button on a tag to be taken to the delete confirmation page
6. Click Delete and verify that the tag was deleted
